Sources/KnitCodeGen/TypeSafetySourceFile.swift

```diff
@@ -160,6 +161,45 @@ public enum TypeSafetySourceFile {
         )
     }
 
+    private static func makePerformanceExtension(config: Configuration) throws -> ExtensionDeclSyntax {
+        let accessorBlock: AccessorBlockSyntax
+        let isAutoInit: Bool
+        if config.assemblyType == .abstractAssembly {
+            accessorBlock = AccessorBlockSyntax(
+                accessors: .getter(.init(stringLiteral: "[.autoInit, .abstract]"))
```

Comment:
I don't see this case covered in the unit tests currently, mind adding a test case for it?